modules/common/services/desktop.nix

```diff
@@ -40,17 +40,6 @@ in
       profiles.graphics.compositor = "labwc";
       graphics = {
         launchers =
-          let
```

Comment:
They might be needed for ctrl-panel as well, at least ´--notls´ option

Reply:
Yes, I can add a wrapper for ctrl-panel as well